scripts/apps/ingest/directives/IngestSourcesContent.js

```diff
@@ -102,6 +114,12 @@ export function IngestSourcesContent(ingestSources, gettext, notify, api, $locat
                                 $options: '-i'},
                             },
                         ]};
+                    } else if ($scope.activeStatusFilter.id !== 'all') {
```

Comment:
It should be possible to filter ingest sources by name and status at the same time. It should also work well with pagination.

Reply:
If we want to filter sources by name then by default status should be "active" or "all"?